megazords/ios-rust/ios/Generated/gleanFFI.modulemap

```diff
@@ -0,0 +1,6 @@
+// This file was autogenerated by some hot garbage in the `uniffi` crate.
```

Comment:
ditto on generated file, (and I guess the `.h` file too)

Reply:
Ditto on this shouldn't be here and may just be leftover from playing with getting the TestServices to build